app.rb

```diff
@@ -67,14 +69,15 @@ def add_game
     new_game.add_author(@authors[author_chosen - 1])
     @games << new_game
     puts 'The game has been added successfully!'
+    save_games(@games)
```

Comment:
I see that you're saving the games everytime that a new one is added.
It seems definitely better than saving the data when the app is quit. Great job here! 🍔